extensions/blocks/calendly/edit.js

```diff
@@ -150,13 +151,7 @@ export default function CalendlyEdit( { attributes, className, clientId, setAttr
 		</>
 	);
 
-	const submitButtonPreview = (
-		<SubmitButton
-			submitButtonText={ submitButtonText }
-			attributes={ attributes }
-			setAttributes={ setAttributes }
-		/>
-	);
+	const submitButtonPreview = <SubmitButton { ...props } />;
```

Comment:
It might be better to just pass the props that are needed? (https://codeburst.io/react-anti-pattern-jsx-spread-attributes-59d1dd53677f)

What do you think?

Reply:
Shame on you linking to Medium 😉 

Yes, it did strike me as a bit lazy, copied it from one of the other uses.